app/routes.py

```diff
@@ -1 +1,302 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, make_response, request, jsonify, abort
```

Comment:
👍🏾 These imports help a great deal when it comes to our request & response cycle